python/py-buildbot-www/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-buildbot-www
+version             2.3.1
+revision            0
+
+maintainers         nomaintainer
+description         Buildbot UI
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://buildbot.net/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             
+
+checksums           sha256  ff8546b7769f156f06e418cdd5621c78086924dbc81d48fbe35f4c98c7e1eb79 \
+                    rmd160  6a31272a9446a57b65b793c329e6d46fb48e1022 \
+                    size    721159
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-buildbot \
+                    port:py${python.version}-buildbot-pkg \
+                    port:py${python.version}-mock \
+                    port:py${python.version}-setuptools
+    
+    depends_lib-append \
+                    port:py${python.version}-treq
```

Comment:
Where is `py-treq` being used?

Reply:
I see that it is declared as a test dependency of `buildbot` itself (not inside the `buildbot-www`), but it's also some semi-optional runtime dependency. I would be inclined to declare that dependency on `py-buildbot`.